packages/dashboard/src/CdcDashboard.tsx

```diff
@@ -29,170 +29,27 @@ import CdcFilteredText from '@cdc/filtered-text'
 import Grid from './components/Grid'
 import Header, { FilterBehavior } from './components/Header'
 import defaults from './data/initial-state'
-import Widget from './components/Widget'
 import DataTable from './components/DataTable'
 import MediaControls from '@cdc/core/components/MediaControls'
 
 import './scss/main.scss'
 import '@cdc/core/styles/v2/main.scss'
```

Comment:
It looks like you removed AdvancedEditor here. While the apply button doesn't work, I like having this in if I need to copy a specific example of a Dashboard to my local. Can we keep this in?

Reply:
I moved the VisualizationsPanel out to its own file as it's a complete component. AdvancedEditor was the only used in there so it was removed from the imports here as Dashboard doesn't use it.